README.rst

```diff
@@ -28,12 +28,22 @@ How can I contribute?
 What do I need to contribute?
 -----------------------------
 
-* A host with python 3.6+, gcc and the python development header files
```

Comment:
Are you sure we don't need gcc anymore for any of the dependencies?

Reply:
Yes I am, we tested this recently. See https://progress.opensuse.org/issues/168706#note-14 (Ignore `jq` and `skopeo` with are only needed in openQA)